src/Bicep.Core/Configuration/ExperimentalFeaturesEnabled.cs

```diff
@@ -16,7 +16,8 @@ public record ExperimentalFeaturesEnabled(
     bool UserDefinedFunctions,
     bool PrettyPrinting,
     bool TestFramework,
-    bool Assertions)
+    bool Assertions,
+    bool DynamicTypeLoadingEnabled)
```

Comment:
nit: The `ExperimentalFeaturesEnabled` class name already has the suffix `Enabled`, so we don't need to duplicate that in the property name.

```suggestion
    bool DynamicTypeLoading)
```